package.json

```diff
@@ -53,6 +54,11 @@
     "simple-git-hooks": "^2.8.0",
     "typescript": "~4.6.3"
   },
+  "devDependencies": {
```

Comment:
Hmm. Should these be in `dependencies`? Or should all dependencies be in `devDependencies`? They're kinda all `devDependencies` in the root.

Reply:
Yes, that's true, and apparently I just followed the same habit that I usually do for public packages. Which tells me everything should probably be in `devDependencies`. I can make that change in another PR but move this to `dependencies` for now.

Reply:
Sounds good!